File: routeMap/app/src/main/java/ru/android/zheka/vm/IGeoVM.java
package ru.android.zheka.vm;

import ru.android.zheka.coreUI.IVM;
import ru.android.zheka.model.IGeoModel;

public interface IGeoVM extends IVM<IGeoModel> {
    void home();
    void points();
    void saveTrace();
    void map();
    void addWayPoints();
}

File: routeMap/app/src/main/java/ru/android/zheka/vm/IGeoVM.kt
package ru.android.zheka.vm

import ru.android.zheka.coreUI.IVM
import ru.android.zheka.model.IGeoModel

interface IGeoVM : IVM<IGeoModel?> {
    fun home()
    fun savePoint()
    fun map()
    fun addCPoint()
    fun goPosition()
    fun hide()
}